great_expectations/profile/user_configurable_profiler.py

```diff
@@ -118,12 +120,19 @@ def __init__(
                 MetricConfiguration("table.columns", {})
             )
         elif isinstance(self.profile_dataset, Validator):
+            context = self.profile_dataset.data_context
             self.all_table_columns = self.profile_dataset.get_metric(
                 MetricConfiguration("table.columns", {})
             )
         else:
             self.all_table_columns = self.profile_dataset.get_table_columns()
 
+        # Check to see if the user has disabled progress bars
+        if context:
+            self._enable_progress_bars = context.progress_bars.is_enabled("profilers")
+        else:
+            self._enable_progress_bars = True
```

Comment:
Opted to make this attr `enable...` since it's cleaner to read. The negation happens when it is actually used.
Tried to adhere to this: http://davej.io/2018/05/negative-naming.html